VideoLocker/src/main/java/org/edx/mobile/view/CourseDiscussionPostsThreadFragment.java

```diff
@@ -300,34 +332,34 @@ protected void onException(Exception ex) {
     }
 
     private void checkNoResultView(EmptyQueryResultsFor query) {
```

Comment:
This doesn't seem like the most descriptive name; maybe change it to `setNoResultView()`?